client/src/components/CodeBlock.js

```diff
@@ -2,18 +2,18 @@ import React, { PropTypes, Components } from 'react';
 import CodeLine from './CodeLine';
 import { connect } from 'react-redux';
 
-// TODO Create styles (indentation, colors, etc.) 
-const CodeBlock = ({ className, codeLines, clickAction, selectedLines }) => (
+const CodeBlock = ({ className, colorClass, codeLines, clickAction, selectedLines }) => (
 			<div>
 				{
 					Object.keys(codeLines).map((key) =>
 						<CodeLine 
 							key={key}
+							clickAction={clickAction}
 							line={codeLines[key].line}
 							lineNumber={codeLines[key].lineNumber}
-							clickAction={clickAction}
-							highlight={selectedLines[selectedLines.length - 1]}
-							onClick={clickAction}
+							colorClass= {colorClass ? colorClass : codeLines[codeLines[key].lineNumber - 1].color}
```

Comment:
Sorry that this seems hacky, but I didn't want to create a new CodeBlock or CodeLine component or pass in too many arbitrary variables.